PowerPointLabs/PowerPointLabs/PictureSlidesLab/PictureSlidesLab.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PowerPointLabs.PictureSlidesLab
+{
+    internal static class PictureSlidesLab
+    {
+#pragma warning disable 0618
```

Comment:
Is this necessary for the code to work?

Reply:
Not really sure about this, removing this and running pptlabs does not seem to cause problems. However, in the TimerLab implementation of this, there was this line.